app.js

```diff
@@ -1,8 +1,23 @@
-function hello(name) {
-  const greeting = `hello ${name}`;
+const { bootstrap } = require("@kaholo/plugin-library");
+
+async function hello(params) {
+  const name = params.helloName || "there";
+  const saySecret = params.saySecret || undefined;
+  const secret = params.secret || undefined;
```

Comment:
Let's not provide default values here, unless it's strongly justified (like Kubectl). `config.json` is the place to do that, and let's promote such approach.

You could also destructure the `params`.

```suggestion
  const {
      helloName,
      saySecret,
      secret,
  } = params;
```

Reply:
Good idea.